include/bitcoin/bitcoin/wallet/qrcode.hpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2011-2015 libbitcoin developers (see AUTHORS)
+ *
+ * This file is part of libbitcoin.
+ *
+ * libbitcoin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License with
+ * additional permissions to the one published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version. For more information see LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef LIBBITCOIN_QRENCODE_HPP
+#define LIBBITCOIN_QRENCODE_HPP
+
+#include <cstddef>
+#include <cstdint>
+#include <iostream>
+#include <bitcoin/bitcoin/compat.hpp>
+#include <bitcoin/bitcoin/define.hpp>
+#include <bitcoin/bitcoin/utility/data.hpp>
+
+#ifdef WITH_QRENCODE
+#include "qrencode.h"
+#endif
+
+
+namespace libbitcoin {
+namespace wallet {
+
+#ifdef WITH_QRENCODE
+
+typedef uint32_t qr_version;
+typedef QRecLevel qr_rec_level;
+typedef QRencodeMode qr_encode_mode;
+typedef uint32_t qr_case_sensitive;
```

Comment:
I don't see the rationale for using the `qr_` prefix in the `bc` namespace as opposed to using the `bc::qr` namespace as below. If the objective is the avoidance of naming conflicts this approach is inconsistent with libbitcoin style.